src/commands/uploadProguard.js

```diff
@@ -0,0 +1,68 @@
+import { createReadStream, statSync } from 'fs';
+import { apiClient } from '../apiClient.js';
+import { formatError } from '../formatError.js';
+import { uploadBuilderBuilder } from '../uploadBuilderBuilder.js';
+import { basename, join } from 'path';
+import { cwd } from 'process';
+import glob from 'glob';
+
+export const command = 'upload-proguard <path>';
+export const describe = 'Upload ProGuard for an android release';
+export const builder = uploadBuilderBuilder('-proguard');
+
+export const handler = async (args) => {
+  const { path, release, apikey, apihost, verbose } = args;
+
+  console.info(`Preparing to upload ProGuard mapping for release ${release} ...`);
+
+  const client = apiClient({ apikey, apihost });
+  await client.checkStatus();
+
+  if (args['gcs-token']) {
+    client.setGCSData({
+      gcsToken: args['gcs-token'],
+      gcsBucket: args['gcs-bucket'],
+    });
+  }
+
+  let proguardPath;
+  if (basename(path) === 'mapping.txt') {
+    proguardPath = path;
+  } else {
+    const workingDirectory = join(cwd(), path);
```

Comment:
Currently if I pass a path to a specific file that doesn't happen to be named `mapping.txt`, the script failes on this line - I mentioned this in the other comment but what would you think about allowing any text file here and renaming it to `mapping.txt` on upload?